cassandane/Cassandane/Cyrus/Conversations.pm

```diff
@@ -380,6 +388,22 @@ sub test_reconstruct_splitconv
     $exp{"A20"}->set_attributes(cid => $exp{"A20"}->make_cid(), basecid => $exp{A}->make_cid());
 
     $self->check_messages(\%exp, keyed_on => 'uid');
+    $talk->select("foo");
+    $self->check_messages(\%exp, keyed_on => 'uid');
+    $talk->select("INBOX");
+
+    # zero everything out
+    $self->{instance}->run_command({ cyrus => 1 }, 'ctl_conversationsdb', '-z', 'cassandane');
+
+    $self->{instance}->run_command({ cyrus => 1 }, 'ctl_conversationsdb', '-d', 'cassandane');
```

Comment:
no need to dump here?

Reply:
Oh yeah, that was just so I could read the output in `-v -v`, I'll remove it.